src/ast/nodes/shared/FunctionBase.ts

```diff
@@ -317,10 +316,7 @@ export default abstract class FunctionBase extends NodeBase {
 
 	private functionParametersOptimized = false;
 	include(context: InclusionContext, includeChildrenRecursively: IncludeChildren): void {
-		const isIIFE =
-			this.parent.type === NodeType.CallExpression &&
-			(this.parent as CallExpression).callee === this;
-		const shoulOptimizeFunctionParameters = isIIFE || this.onlyFunctionCallUsed();
+		const shoulOptimizeFunctionParameters = this.onlyFunctionCallUsed();
```

Comment:
Hey, I can contribute a spelling fix, at least 😉 

```suggestion
		const shouldOptimizeFunctionParameters = this.onlyFunctionCallUsed();
```